### PR TITLE
User lifecycle

### PR DESCRIPTION
**Pull request by kpslp**

### CODE REVIEW

**Review by BananZG — APPROVED** [review]

File: docs/user-lifecycle.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+This document outlines the rules for managing user accounts within the TechPass system. These rules apply to all user accounts unless specific conditions are met that overwrite the general rules.
+
+## General rules
+
+### When a new user is created or invited
+
+1. **No sign-in within 23 days from creation date**:
+   - We send daily reminders to the user that their account will be terminated.
+   - If the user does not sign in by day 30, we stop reminders and terminate the user account.
```
Comment (yilunyou) [inline docs/user-lifecycle.md]:
```suggestion
   - If the user does not sign in by **day 30**, we stop reminders and terminate the user account.
```

File: docs/user-lifecycle.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+   - Five days after termination, the user account is deleted.
+
+### If the user has signed in before
+
+1. **No new sign-in in the past 60 days**:
+   - We send daily reminders to the user that their account will be disabled.
+   - If the user does not sign in by day 90, we stop reminders and disable the user account.
```
Comment (yilunyou) [inline docs/user-lifecycle.md]:
```suggestion
   - If the user does not sign in by **day 90**, we stop reminders and disable the user account.
```

File: docs/user-lifecycle.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+
+2. **No new sign-in in the past 173 days**:
+   - We send daily reminders to the user that their account will be terminated.
+   - If the user does not sign in by day 180, we stop reminders and terminate the user account.
```
Comment (yilunyou) [inline docs/user-lifecycle.md]:
```suggestion
   - If the user does not sign in by **day 180**, we stop reminders and terminate the user account.
```

File: docs/user-lifecycle.md (hunk 1 above)
Comment (yilunyou) [reply docs/user-lifecycle.md]: can emphasize on the days where destructive actions happen

File: docs/user-lifecycle.md [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+
+In certain cases, account lifecycle events are triggered manually, especially when a user leaves the organisation. The process differs depending on the user type:
+
+### Public officers
+
+- Public officers are disabled, blocked, or deleted from the Whole of Government Azure Active Directory (WoG AAD) based on an event from CAM.
+
+### Vendors
+
+- Vendor accounts are managed by tenant admins and can be disabled, blocked, or deleted via the portal, service requests, or from CAM.
```
Comment (yilunyou) [inline docs/user-lifecycle.md]:
```suggestion
- Vendor accounts can be disabled, blocked, or deleted via service requests or from CAM.
```

we plan to remove the ability to manage accounts by tenant admins

**Review by yilunyou — APPROVED** [review]
@BananZG please apply the changes. and i will approve again for merge

**Pull request merged**